composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ongr/elasticsearch-dsl",
+    "name": "gbourgeat/elasticsearch-dsl",
```

Comment:
Remove your prefix in the package-name in composer.json

Reply:
thanks @dannydorfel for taking a look at the diff